modules/@apostrophecms/asset/lib/webpack/apos/webpack.config.js

```diff
@@ -64,7 +65,7 @@ module.exports = ({
     resolve: {
       extensions: [ '.*', '.js', '.vue', '.json' ],
       alias: {
-        vue$: '@vue/runtime-dom',
+        vue$: getVueAlias(),
```

Comment:
```suggestion
        vue$: getVueAlias(config),
```

Reply:
We're already in the config definition, don't see the point here.